__mocks__/aws-sdk/clients/devicefarm.js

```diff
@@ -1,7 +1,7 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 const responses = {
-  pass: () => Promise.resolve({ url: 'http://localhost:4444' }),
+  pass: () => Promise.resolve({ url: 'http://localhost:4444/devicefarm-test-grid' }),
```

Comment:
Updated the return value to be extra sure we hit this function and not our `http://localhost:4444` default